### PR TITLE
fix(cli): display the current directory when cloning into current dir

### PR DESCRIPTION
**Pull request by uncenter**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [x] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

The success message after cloning a template into the current directory is a little confusing:

Screenshot 2023-12-23 at 15 49 58 (kitty)

Where was it cloned? Did it work?

Not sure if this is the ideal solution for y'all but with this PR it looks like this instead:

Screenshot 2023-12-23 at 15 52 52 (kitty)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -74,7 +74,10 @@ const mainCommand = defineCommand({
     });
 
     const _from = r.name || r.url;
-    const _to = relative(process.cwd(), r.dir);
+    const _to =
+      process.cwd() === r.dir
+        ? "the current directory"
```
Comment (pi0) [inline src/cli.ts:79]: Maybe show the full path in this situation to avoid confusion? ideas welcome

Comment (uncenter) [reply src/cli.ts:79]: Maybe we just display the entire path all the time?

Comment (pi0) [reply src/cli.ts:79]: Using relative is to make CLI output shorter. And relative paths actually are valid within the context of current shell.

Current dir's relative path can be also shown as `.` or `./` btw I think second is also readable but full path could be more in this exceptional case as not every starter user probably knows about it.

**Comment by uncenter** [comment]
As you suggested, looks like this now:

Screenshot 2023-12-23 at 16 14 56 (kitty)

File: src/cli.ts [hunk 2]
```diff
@@ -74,7 +74,7 @@ const mainCommand = defineCommand({
     });
 
     const _from = r.name || r.url;
-    const _to = relative(process.cwd(), r.dir);
+    const _to = process.cwd() === r.dir ? "./" : relative(process.cwd(), r.dir);
```
Comment (pi0) [inline src/cli.ts:77]: Refactor: Can you verify if we could instead use this:

```suggestion
    const _to = relative(process.cwd(), r.dir) || "./";
```

**Comment by uncenter** [comment]
Yep, that works. One less `process.cwd()` call!

**Review by pi0 — APPROVED** [review]
Thanks!

**Pull request merged**